src/ucs/async/async.c

```diff
@@ -382,7 +382,8 @@ void ucs_async_context_destroy(ucs_async_context_t *async)
 static ucs_status_t
 ucs_async_alloc_handler(int min_id, int max_id, ucs_async_mode_t mode,
                         int events, ucs_async_event_cb_t cb, void *arg,
-                        ucs_async_context_t *async, int *id_p)
+                        ucs_async_context_t *async, int *id_p,
+                        int **timer_id_p)
```

Comment:
can we return ucs_async_handler_t** handler_p here, instead of both id_p and timer_id_p?